### PR TITLE
Implemented NM-17: Add button and scripts for work with requests on

### PR DESCRIPTION
**Pull request by HomerJS**
- Add button to all type of product
- Add modal window

### CODE REVIEW

File: app/design/frontend/base/default/layout/BeC_Notify.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <reference name="product.info">
+            <block type="notify/button" name="product.info.availability" as="product_type_availability" template="becnotify/button.phtml" />
+        </reference>
+    </PRODUCT_TYPE_configurable>
+    <PRODUCT_TYPE_virtual>
+        <reference name="product.info">
+            <block type="notify/button" name="product.info.availability" as="product_type_availability" template="becnotify/button.phtml" />
+        </reference>
+    </PRODUCT_TYPE_virtual>
+
```
Comment (HomerJS) [inline app/design/frontend/base/default/layout/BeC_Notify.xml]: oops. Fixed

File: app/design/frontend/base/default/layout/BeC_Notify.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    
+    <catalog_product_view>
+        <reference name="before_body_end">
+            <reference name="head">
```
Comment (HomerJS) [inline app/design/frontend/base/default/layout/BeC_Notify.xml]: Fixed

File: app/design/frontend/base/default/template/becnotify/button.phtml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * @category    design
+ * @package     base_default
+ * @license     http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ */
+?>
+
+<input class="button becnotifyme-button" data-id_product = "<?php  echo $this->getProductId(); ?>" type="submit" value="<?php echo $this->__('Notify me button');?>">
```
Comment (HomerJS) [inline app/design/frontend/base/default/template/becnotify/button.phtml]: Fixed. 
- Add in block "Button.php" - constructor.
- Delete method from block "Button.php"
- Change method in "button.phtml"

File: app/code/local/BeC/Notify/Block/Requests.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magento.com for more information.
+ *
+ * @category    Mage
+ * @package     Mage_Checkout
```
Comment (aandolg) [inline app/code/local/BeC/Notify/Block/Requests.php]: I think block text license and description package checkout must  be deleted.

Comment (HomerJS) [inline app/code/local/BeC/Notify/Block/Requests.php]: Fixed/

**Comment by HomerJS** [comment]
Add button to bundle product

**Pull request merged**